packages/elements-core/src/components/NonIdealState.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { Box, Flex, Heading, Icon, Text } from '@stoplight/mosaic';
+import * as React from 'react';
+
+type NonIdealStateProps = {
+  description?: React.ReactChild;
+  icon?: IconProp;
+  title?: React.ReactNode;
```

Comment:
Child or Node? 😉 IDK which one's better but let's be consistent.

Reply:
That comes from original ui-kit component, I just used the same props. But yes we can have it unified